src/Uno.UWP/Storage/Pickers/FileOpenPicker.Android.cs

```diff
@@ -1,166 +1,166 @@
-			return files.FirstOrDefault();
-		}
-
-		private async Task<IReadOnlyList<StorageFile>> PickMultipleFilesTaskAsync(CancellationToken token)
-		{
-			return await PickFilesAsync(true, token);
-		}
-
-		private async Task<FilePickerSelectedFilesArray> PickFilesAsync(bool multiple, CancellationToken token)
-		{
-			if (!(ContextHelper.Current is Activity appActivity))
-			{
-				throw new InvalidOperationException("Application activity is not yet set, API called too early.");
-			}
-
-			if (Android.OS.Build.VERSION.SdkInt < Android.OS.BuildVersionCodes.Kitkat)
-			{
-				throw new NotSupportedException("FileOpenPicker requires Android KitKat (API level 19) or newer");
-			}
-
-			var action = Intent.ActionOpenDocument;
-
-			var intent = new Intent(action);
-			intent.PutExtra(Intent.ExtraAllowMultiple, multiple);
-
-			var settingName = string.Format(StorageIdentifierFormatString, SettingsIdentifier);
-			if (ApplicationData.Current.LocalSettings.Values.ContainsKey(settingName))
-			{
-				var uri = ApplicationData.Current.LocalSettings.Values[settingName].ToString();
-				intent.PutExtra(Android.Provider.DocumentsContract.ExtraInitialUri, uri);
-			}
-
-			intent.SetType("*/*");
-
-			var mimeTypes = GetMimeTypes();
-			intent.PutExtra(Intent.ExtraMimeTypes, mimeTypes);
-
-			_currentFileOpenPickerRequest = new TaskCompletionSource<Intent?>();
-
-			appActivity.StartActivityForResult(intent, RequestCode);
-
-			var resultIntent = await _currentFileOpenPickerRequest.Task;
-			_currentFileOpenPickerRequest = null;
-
-			if (resultIntent?.ClipData != null)
-			{
-				List<StorageFile> files = new List<StorageFile>();
-				bool wasPath = false;
-
-				for (var i = 0; i < resultIntent.ClipData.ItemCount; i++)
-				{
-					var item = resultIntent.ClipData.GetItemAt(i);
-					if (item?.Uri == null)
-					{
-						continue;
-					}
-					var file = StorageFile.GetFromSafUri(item.Uri);
-					files.Add(file);
-
-					// for PickMultipleFilesAsync(), we preserve (existing) path of last selected file
-					if (!string.IsNullOrEmpty(file.Path))
-					{
-						ApplicationData.Current.LocalSettings.Values[settingName] = file.Path;
-						wasPath = true;
-					}
-				}
-
-				if(!wasPath)
-				{   // if we have no path in any of files, remove setting - next call to Picker will not have InitialDir
-					ApplicationData.Current.LocalSettings.Values.Remove(settingName);
-				}
-				return new FilePickerSelectedFilesArray(files.ToArray());
-			}
-			else if (resultIntent?.Data != null)
-			{
-				var file = StorageFile.GetFromSafUri(resultIntent.Data);
-				return new FilePickerSelectedFilesArray(new[] { file });
-			}
-
-			return FilePickerSelectedFilesArray.Empty;
-		}
-
-		private string[] GetMimeTypes()
-		{
-			if (FileTypeFilter.Contains("*"))
-			{
-				return new[] { "*/*" };
-			}
-
-			List<string> mimeTypes = new List<string>();
-
-			using Android.Webkit.MimeTypeMap? mimeTypeMap = Android.Webkit.MimeTypeMap.Singleton;
+﻿#nullable enable

```

Comment:
I modified directly in GitHub, and it looks like it changed the line ending (I think from LF to CRLF). Hope that's fine?

Reply:
That's fine, though we'll probably need to find out if there's a way to change the configuration to avoid it in the future. Did you use the github.dev? or the web "textbox-like" experience ?

Reply:
@jeromelaban The regular textbox editor in GitHub.com